charts/riff-streaming-runtime/values.yaml

```diff
@@ -0,0 +1,3 @@
+---
+keda:
+  enabled: false
```

Comment:
Is that really what we want? We want keda as part of streaming in all cases (unless very explicitly disabled because you have your own keda), not just when installing the big-riff chart, don't we?

Reply:
this is consistent with other charts. For the individual charts we disable all of the nested dependencies to make it easier to manage a single chart in isolation. Maybe keda is already installed for some reason.

In the `riff` chart, we enabled keda by default when the streaming runtime is enabled (each runtime is disabled by default).